npcx_i2c_ctrl_transfer does not match original declaration

asemjonovs opened the pull request:
Fix npcx_i2c_ctrl_transfer arguments to match header to prevent warning: type of 'npcx_i2c_ctrl_transfer' does not match original declaration

ChiHuaL reviewed (approved):
Thanks for helping fix this.

fabiobaltieri commented:
@asemjonovs I think this needs a rebase to fix the CI error, can you rebase and force push please?

asemjonovs committed:
i2c: npcx_i2c_ctrl_transfer does not match original declaration

Fix npcx_i2c_ctrl_transfer arguments to match header to prevent
warning: type of 'npcx_i2c_ctrl_transfer' does not match original
declaration

Signed-off-by: Al Semjonovs <asemjonovs@google.com>
